opts/parse.go

```diff
@@ -23,14 +23,17 @@ func ReadKVEnvStrings(files []string, override []string) ([]string, error) {
 }
 
 func readKVStrings(files []string, override []string, emptyFn func(string) (string, bool)) ([]string, error) {
-	variables := []string{}
+	var variables []string
 	for _, ef := range files {
 		parsedVars, err := parseKeyValueFile(ef, emptyFn)
 		if err != nil {
 			return nil, err
 		}
 		variables = append(variables, parsedVars...)
 	}
+	if len(variables) == 0 {
```

Comment:
Question: Why not just return the `append` which would be no-op?
```go
var variables []string
for _, ef := range files {
...
}
return append(variables, override...), nil
```

Reply:
Basically did that to prevent the value from changing (`[]string{}` changing to `nil` or vice-versa).